### PR TITLE
Added support of --json timestamp in python client.py

### PR DESCRIPTION
**Pull request by aayagar001**
Modified client.py to add function for converting metadata to timestamp of each word

### CODE REVIEW

**Comment by hamidelmaazouz** [comment]
aayagar001 <notifications@github.com> schrieb am Mi., 13. Nov. 2019, 07:05:

**Comment by aayagar001** [comment]
Hi @lissyx I tried implementing the code by seeing the c++ code available in client.cc in native client (https://github.com/mozilla/DeepSpeech/blob/master/native_client/client.cc#L293)
I wanted the time stamp info to be available in python code, metadata info was already available so thought just a post processing similar to c++ code will do. 
Let me know if anything more is needed.

**Comment by lissyx** [comment]
> Hi @lissyx I tried implementing the code by seeing the c++ code available in client.cc in native client (https://github.com/mozilla/DeepSpeech/blob/master/native_client/client.cc#L293)
> I wanted the time stamp info to be available in python code, metadata info was already available so thought just a post processing similar to c++ code will do.
> Let me know if anything more is needed.

Right, but **why** do you want that information exposed from this tool and as JSON ? Are you doing some processing after in your workflow ?

**Comment by aayagar001** [comment]
Yes I wanted to create a python API for subtitle kind of thing so I needed this in python . I used the deepspeech python package programatically in my code with help of client.py and found missing implementation there for timestamp so thought of adding the same. Hope this clarifies.

**Comment by lissyx** [comment]
> Yes I wanted to create a python API for subtitle kind of thing so I needed this in python . I used the deepspeech python package programatically in my code with help of client.py and found missing implementation there for timestamp so thought of adding the same. Hope this clarifies.

Sorry, it's still not clear in my mind. Can you explain more your workflow ? How does `client.py` plays a role in your code ?

i'm just worried you are doing something the wrong way in your system, and whether this code should really be in `client.py` or just another Python example.

**Comment by lissyx** [comment]
So please @aayagar001 can you guide me a bit more into your workflow ? There's no good reason you **need** to do changes to `client.py`.

**Comment by aayagar001** [comment]
Yes you pointed right that it can be another python example. I made changes in client.py looking forward that it might help others to work in getting timestamp info which currently was not there as example for python and many might refer this client.py as an example to use in thier python code.
I am new to this so if you feel that it should not be there then it's fine.

**Comment by lissyx** [comment]
>  I am new to this so if you feel that it should not be there then it's fine.

Well, there could be legit reasons as to push that, being an example / on-par with the C++ can be a good one. I just want to ensure you are not doing this for other reasons that may be wrong.

**Comment by aayagar001** [comment]
No other reasons other than just providing code example for timestamp extraction from metadata, as many refer client.py as an example to implement it in python code and found many posts on [discourse](https://discourse.mozilla.org/t/native-client-metadata/41199/17) around people asking for timestamp in python

**Comment by lissyx** [comment]
Thanks @aayagar001 !

File: native_client/python/client.py [hunk 1]
```diff
@@ -62,6 +104,8 @@ def main():
                         help='Print version and exits')
     parser.add_argument('--extended', required=False, action='store_true',
                         help='Output string from extended metadata')
+    parser.add_argument('--json', required=False, action='store_false',
```
Comment (lissyx) [inline native_client/python/client.py]: I missed this, `store_false` makes that a default-enabled parameter.

Comment (lissyx) [reply native_client/python/client.py]: @aayagar001 can you send another pr that switches that to store_true?

Comment (aayagar001) [reply native_client/python/client.py]: Sure.

Comment (aayagar001) [reply native_client/python/client.py]: Created PR #2546

**Pull request merged**